packages/evolution-generator/src/scripts/generate_choices.py

```diff
@@ -108,11 +117,11 @@ def generate_choices(input_file: str, output_file: str):
                         f"{INDENT}{INDENT}label: {{\n"
                         f"{INDENT}{INDENT}{INDENT}fr: '{choice['label']['fr']}',\n"
                         f"{INDENT}{INDENT}{INDENT}en: '{choice['label']['en']}'\n"
-                        f"{INDENT}{INDENT}}}{',' if choice['conditional'] else ''}\n"
+                        f"{INDENT}{INDENT}}}{',' if "conditional" in choice and choice['conditional'] is not None else ''}\n"
```

Comment:
Je crois que c'est un endroit où tu pourrais faire `choice.get('conditional'` au lieu de `"conditional" in choice and choice['conditional']` tel que suggéré dans #548

Reply:
Ah ouais, c'est modifié.